modules/services/eks/iam.tf

```diff
@@ -0,0 +1,72 @@
+// This is a Single Account installation. The resources are created globally (instead of regionally). 
+data "aws_iam_policy_document" "ecr_pull_image" {
+  count = local.n
+
+  statement {
+    sid = "SysdigEcrPullImagePermissions"
+
+    effect = "Allow"
+
+    actions = [
+      "ecr:GetDownloadUrlForLayer",
+      "ecr:BatchGetImage",
+      "ecr:BatchCheckLayerAvailability",
+      "ecr:ListImages",
+      "ecr:GetAuthorizationToken",
+    ]
+
+    resources = [
+      "*",
+    ]
+  }
+}
+
+resource "aws_iam_policy" "ecr_pull_image" {
+  count = local.n
+
+  name        = var.ecr_role_name
+  description = "Allows Sysdig Secure to pull ECR images"
+  policy      = data.aws_iam_policy_document.ecr_pull_image[0].json
+  tags        = var.tags
+}
+
+data "aws_iam_policy_document" "ecr_assume_role" {
+  count = local.n
```

Comment:
Do we actually need this data to be conditional with count? We can simplify the usage by not specifying a `count`, since it doesn't create any resource.